src/test/java/io/managed/services/test/registry/RegistryKafkaIntegrationTest.java

```diff
@@ -60,9 +60,9 @@
 public class RegistryKafkaIntegrationTest extends TestBase {
     private static final Logger LOGGER = LogManager.getLogger(RegistryKafkaIntegrationTest.class);
 
```

Comment:
currently only place for 1 service regisstry instance, one test will fail at the moment.